src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.CrossTargeting.targets

```diff
@@ -24,7 +24,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   ============================================================
    -->
   <Target Name="Publish">
-    <Error Text="The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, please specify the framework for the published application." />
+    <Error Text="The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, you must specify the framework for the published application." />
```

Comment:
Looks like we have an error message here that isn't localized.  Can we fix that?

(It can either be in this PR or we can file an issue to fix it later).

Reply:
Can you reference localized strings from .targets files?

Reply:
We use the `SdkError` task to support localization for errors.